packages/devextreme/js/localization/messages/vi.json

```diff
@@ -195,8 +195,8 @@
         "dxDataGrid-masterDetail": "Cell with details",
 
         "dxTreeList-ariaTreeList": "Tree list with {0} rows and {1} columns",
-        "dxTreeList-ariaSearchInGrid": "Search in the tree list",
         "dxTreeList-ariaExpandableInstruction": "Press Ctrl + right arrow to expand the focused node and Ctrl + left arrow to collapse it",
+        "dxTreeList-ariaSearchInGrid": "Search in the tree list",
```

Comment:
excess change

Reply:
This is necessary... this is added by auto-generation of localization messages. once remove its cause some CI issues

![image](https://github.com/user-attachments/assets/534b3f12-6b3a-4d5c-81c7-63c8328a35db)

Reply:
https://github.com/DevExpress/DevExtreme/actions/runs/10697916763/job/29656375230?pr=27994#step:3:578

Reply:
[be6adb7](https://github.com/DevExpress/DevExtreme/pull/27994/commits/be6adb77d42d7943cc6cf6f269925de143058e28)

Reply:
okay, i see that it other files the order is how it's updated in this one.
Do idea why previously it was a specific order for this particular locale - maybe sloppiness in previous PRs. 
Anyway it's not so important to pay much attention for this